src/main/webapp/loggedIn.html

```diff
@@ -9,9 +9,8 @@
     <!-- Initialize firebase using CDN -->
     <script src="https://www.gstatic.com/firebasejs/ui/4.6.1/firebase-ui-auth.js"></script>
     <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/4.6.1/firebase-ui-auth.css" />
-    <script src="common.js"></script>
     <link href="style.css" rel="stylesheet" type="text/css" media="screen" />
-    <script src="app.js"></script>
+    
   </head>
   <body onload="addMenu()">
```

Comment:
It's generally good practice to put the onload callback in javascript rather than in html.
https://www.javascripttutorial.net/javascript-dom/javascript-onload/

Then you don't need to load the scripts after the body.